### ISSUE TITLE
Problems making plugin available

### ISSUE THREAD

**Issue by knottnhupfer** [stone-payments/kong-plugin-url-rewrite#24]
Hi,

I'm trying to use your plugin with the dockerimage "kong:1.3.0-alpine". In my Dockerfile I install the plugin with:
`RUN luarocks install kong-plugin-url-rewrite`

I'm checking it trough:
curl http://localhost:8001 | python -mjson.tool | jq .plugins

Trying to enable it trough environment variables fails when starting:

`KONG_PLUGINS=url-rewrite`
or
`KONG_PLUGINS=plugin-url-rewrite`

Hope you can help me out.

Thx a lot

/david

**Comment by PFadel** [comment]
Hello @knottnhupfer ! As of now, we only support Kong community edition versions  0.14.x and 0.13.x, you can see more info about this in [here](https://docs.konghq.com/hub/stone-payments/kong-plugin-url-rewrite/) under `Compatibility`.

We would happily accept PR's to improve the range of kong versions we support, if you feel like it. :sweat_smile:

**Comment by knottnhupfer** [comment]
Hi @PFadel ,
thx a lot for your quick response, didn't think about compatibility issues ... sorry for that, I'm really new to kong.
Mhm ... will think about it but kong as also lua is very new territory for me, until yet I didn't have to read any lua code line :sweat_smile:

**Comment by ZVilusinsky** [comment]
So I quess further support for this is plugin is dead and I sould look for another sollutions?

**Comment by PFadel** [comment]
Hi @ZVilusinsky ,
support is not dead, but as it is this works for us here at @stone-payments , so this feature is not in our roadmap.

As I said, we would happily accept PR's to improve the range of kong versions we support, if this is something you'd like to use.

**Comment by ZVilusinsky** [comment]
@PFadel Yeah, sorry, LUA is a no-go (pun not intended, Kong 2+ supports plugins in Go).
